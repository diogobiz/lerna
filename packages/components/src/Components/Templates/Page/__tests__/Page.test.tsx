import React from 'react'
import TestRenderer from 'react-test-renderer'

import { SANThemeProvider, SANThemeCreateTheme } from '../../../../'
import { SANPage } from '../'

it('renders correctly', () => {
    const component = (
        <SANThemeProvider theme={SANThemeCreateTheme({})}>
            <SANPage>Content</SANPage>
        </SANThemeProvider>
    )
    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
