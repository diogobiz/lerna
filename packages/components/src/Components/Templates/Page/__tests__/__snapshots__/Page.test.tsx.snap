// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-EHOje hFFNvO"
  display="flex"
>
  <div
    className="sc-EHOje iMikaY"
  >
    Content
  </div>
</div>
`;
